index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="styles.css">
+<html lang="es">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mi CV</title>
+  </head>
+  <body>
+    <div id="encabezado">
+      <div>
+        <div id="nombre"> 
+          <h1>______________________________</h1>
+          <h1>Isabela Díaz Acosta</h1>
+          <h3>Cra 3#10A02 San Teo II • isadiac06@gmail.com • 3125822792</h3>
+          <h1>______________________________</h1>
+          </div>
+      </div>
+      <div id="img1">
+        <img id="foto-perfil" src="foto-para-cv.jpg">
+      </div>
+    </div>
+    <div id="columnas">
+      <div>
+        <div>
+          <h3>Educación</h3>
+          <p>
+            <b>Colegio Trinidad del monte / Colegio Internacional SEK Colombia</b>
+          </p>
+          <p>Bachillerato Bilingue</p>
+          <p>20 de junio 2023</p>
+          <p><b>Universidad de la Sabana</b></p>
+          <p>Pregrado Ingeniería informática</p>
+          <h3>Educación no formal</h3>
+          <p>
+            <b>Curso de introducción a la inteligencia artificial con Python</b>
+          </p>
+          <p>EDX en colaboración con Harvard</p>
+          <p><b>Estudios de danza clásica y latinoamericana</b></p>
+          <p>
+            Escuela de ballet de Chía Ariadna Páramo - Incolballet - Danzar Colombia
+            - Vasilliev Academy
+          </p>

```

Comment:
Al ser estos elementos categóricos que tienen un sentido jerárquico, lo más correcto es marcarlos como una lista
```suggestion
          <h3>Educación</h3>
          <ul>
             <li>
                 <p>
                    <b>Colegio Trinidad del monte / Colegio Internacional SEK Colombia</b>
                 </p>
             </li>
             
             <li>
                  <p>Bachillerato Bilingue</p>
                  <p>20 de junio 2023</p>
             </li>
          </ul>
         ...
         ...
         ...
```